### ISSUE TITLE
Travis Build Broken

### ISSUE THREAD

**Issue by Wulfheart** [kellydunn/golang-geo#68]
There are problems with the travis build as in #46. Again there is a problem with goose:

```
$ go get -d bitbucket.org/liamstask/goose/lib/goose
package bitbucket.org/liamstask/goose/lib/goose: https://api.bitbucket.org/1.0/repositories/liamstask/goose: 410 Gone
The command "go get -d bitbucket.org/liamstask/goose/lib/goose" failed and exited with 1 during .
Your build has been stopped.
```
Somehow the link is gone on Bitbucket.
Until this is resolved all Travis builds will fail.

**Comment by kellydunn** [comment]
Yep, I believe the new maintainer is `github.com/pressly/goose`.  Will have to investigate the requisite changes to get the builds working again.

**Comment by Wulfheart** [comment]
Goose is still available on bitbucket [here](https://bitbucket.org/liamstask/goose/src/master/). However it seems that the download path is broken. I suspect it is a bitbucket issue as the link does not exist.
`github.com/pressly/goose` has some changes in it.
Might it be viable to fork the actual repository on github so no changes except the URL would apply?
